roles/keystone-defaults/defaults/main.yml

```diff
@@ -60,18 +60,25 @@ keystone:
     sp:
       # Enable keystone to be a service provider
       # for keystone to keystone federation
-      k2k:
-        enabled: False
+      saml:
```

Comment:
Be sure to create a change request for upgrade documentation to indicate what needs too be done to an existing environment on upgrade.